test/unit/mock/objects.js

```diff
@@ -587,6 +587,84 @@ const installationMock = {
   uid: 'installation_uid'
 }
 
+const globalFieldDiff = [{
+  "uid": "global_field_uid",
```

Comment:
@nadeem-cs Js file JSON key should not include quote ex `"uid"` and change double quote to single quote `lint-issue`